metrics-core/src/main/java/software/amazon/swage/metrics/jmx/sensor/DiskUsageSensor.java

```diff
@@ -43,8 +43,10 @@ public TypedMap addContext(final TypedMap existing)
             FileStore fileStore = Files.getFileStore(Paths.get(System.getProperty("user.dir")));
 
             long size = fileStore.getTotalSpace();
-            long gb_size = size/(K*K*K);
-            return ImmutableTypedMap.Builder.from(existing).add(DISK_SIZE, Long.valueOf(gb_size)).build();
+            return ImmutableTypedMap.Builder
```

Comment:
Again if there's an issue with the style here, it should be raised for the whole code base not changed one off.